rules/windows/process_creation/proc_creation_win_msiexec_install_remote.yml

```diff
@@ -0,0 +1,44 @@
+title: Suspicious Msiexec Quiet Install From Remote Location
+id: 8150732a-0c9d-4a99-82b9-9efb9b90c40c
+related:
+    - id: f7b5f842-a6af-4da5-9e95-e32478f3cd2f
+      type: similar
+status: experimental
+description: Detects usage of Msiexec.exe to install packages hosted remotely quietly
+author: Nasreddine Bencherchali
+date: 2022/10/28
+references:
+    - https://www.microsoft.com/en-us/security/blog/2022/10/27/raspberry-robin-worm-part-of-larger-ecosystem-facilitating-pre-ransomware-activity/
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_img:
+        - Image|endswith: '\msiexec.exe'
+        - OriginalFileName: 'msiexec.exe'
+    selection_cli:
+        # Note that there is no space before and after the arguments because it's possible to write a commandline as such
+        # Example: msiexec -q/i [MSI Package]
+        CommandLine|contains:
+            - '/i'
+            - '-i'
+            - '/package'
+            - '-package'
+            - '/a'
+            - '-a'
+            - '/j'
+            - '-j'
+    selection_quiet:
+        CommandLine|contains:
+            - '/q'
+            - '-q'
+    selection_remote:
```

Comment:
keyword search or mlssing field name ?

Reply:
Missing field name. Thanks for the catch